alws/crud/build.py

```diff
@@ -205,8 +206,17 @@ async def remove_build_job(db: Session, build_id: int) -> bool:
         build = build.scalars().first()
         if build is None:
             raise DataNotFoundError(f'Build with {build_id} not found')
+        if build.products:
+            product_names = "\n".join((
+                product.name
+                for product in build.products
+            ))
+            raise BuildError(
+                f"Cannot delete Build={build_id}, "
+                f"build contains in following products:\n{product_names}"
+            )
         if build.released:
-            return False
+            raise BuildError(f"Build with {build_id=} is released")
```

Comment:
doesn't the `=` sign cause troubles when being in this code branch?

Reply:
I'm not sure, but I'll check it

Reply:
![image](https://user-images.githubusercontent.com/64651799/183611798-c1b0fdc7-eb4b-44dc-aa5d-bb011b329547.png)
works fine